packages/TelegramClient-Core.package/TCCSuperGroupChat.class/instance/initializeFromChatEvent..st

```diff
@@ -2,4 +2,4 @@ initialization
 initializeFromChatEvent: anEvent
 
 	super initializeFromChatEvent: anEvent.
-	self groupId: (((anEvent at: 'chat') at: 'type') at: 'supergroup_id')
\ No newline at end of file
+	self groupId: ((anEvent at: 'type') at: 'supergroup_id')
```

Comment:
Missing dot (also in `BasicGroupChat`)
 + 
Maybe define this method on the superclass and let the subclasses overwrite a `groupKey` or something like it.

Reply:
Sounds like a good idea :)